lib/utils/rename.js

```diff
@@ -1,16 +1,182 @@
 'use strict'
 var fs = require('graceful-fs')
+var path = require('path')
+var rimraf = require('rimraf')
```

Comment:
Yes, great suggestion. I probably should have familiarized myself with the rest of the guild directory before jumping in with a rewrite!

What do you think of the rest of this code?